dandi/cli/cmd_validate.py

```diff
@@ -31,32 +32,31 @@
     default="none",
 )
 @click.argument("paths", nargs=-1, type=click.Path(exists=True, dir_okay=True))
+@click.pass_context
 @map_to_click_exceptions
 def validate_bids(
+    ctx,
     paths,
     schema,
     report,
     report_path,
     grouping="none",
 ):
     """Validate BIDS paths.
-
     Notes
     -----
-    Used from bash, eg:
-        dandi validate-bids /my/path
+    * Used from bash, eg:
+    dandi validate-bids /my/path
+    * Deprecated
     """
 
-    from ..validate import validate_bids as validate_bids_
-
-    validator_result = validate_bids_(  # Controller
-        *paths,
-        report=report,
-        report_path=report_path,
-        schema_version=schema,
+    warnings.filterwarnings("default")
```

Comment:
interesting -- without this I was shown no `DeprecationWarning`, odd...  @jwodder - do you know why?

Reply:
yep. took me a while to figure it out as well...